<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemDialog</class>
 <widget class="QDialog" name="itemDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new item</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Content</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="editItem"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>State</string>
     </property>
     <layout class="QVBoxLayout" name="_3">
      <item>
       <widget class="QRadioButton" name="radioNone">
        <property name="text">
         <string>&amp;None</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/empty.svg</normaloff>:/data/images/empty.svg</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioProgress">
        <property name="text">
         <string>&amp;In progress</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/uncheck.svg</normaloff>:/data/images/uncheck.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioFailure">
        <property name="text">
         <string>&amp;Failed</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/failure.svg</normaloff>:/data/images/failure.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSuccess">
        <property name="text">
         <string>&amp;Succeeded</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/check.svg</normaloff>:/data/images/check.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioBasic">
        <property name="text">
         <string>&amp;Basic</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/empty.svg</normaloff>:/data/images/empty.svg</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioNote">
        <property name="text">
         <string>N&amp;ote</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/text.svg</normaloff>:/data/images/text.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSound">
        <property name="text">
         <string>A&amp;udio</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/speaker.svg</normaloff>:/data/images/speaker.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioImage">
        <property name="text">
         <string>I&amp;mage</string>
        </property>
        <property name="icon">
         <iconset resource="../../ressource.qrc">
          <normaloff>:/data/images/image.svg</normaloff>:/data/images/image.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editFile">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolBrowse">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushBrother">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../ressource.qrc">
         <normaloff>:/data/images/add.svg</normaloff>:/data/images/add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushChild">
       <property name="text">
        <string>C&amp;hild</string>
       </property>
       <property name="icon">
        <iconset resource="../../ressource.qrc">
         <normaloff>:/data/images/son.svg</normaloff>:/data/images/son.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../ressource.qrc">
         <normaloff>:/data/images/stop.svg</normaloff>:/data/images/stop.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushCancel</sender>
   <signal>clicked()</signal>
   <receiver>itemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
